__tests__/fixtures/lifecycle-scripts/script-segfault/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "compile": "gcc segfault.c -g -o segfault",
```

Comment:
😮  Will this work on AppVeyor (Windows)

I'm not sure if Windows has the concept of "exiting with a signal" so it may be better to disable this test on Windows.